resources/buildConfigDefinitions.js

```diff
@@ -0,0 +1,221 @@
+function last(array) {
+  return array[array.length - 1];
+}
+
+const parsers = require('../src/Options/parsers');
+
+const letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+function toENV(key) {
+  var str = '';
+  let previousIsUpper = false;
+  for(let i = 0; i < key.length; i++) {
+    const char = key[i];
+    if (letters.indexOf(char) >= 0) {
```

Comment:
this should probably be a function that can be passed to map. (just thinking out loud. if it doesn't appeal to you, don't do it).